wagtail_streamfield_migration_toolkit/autodetect/comparers.py

```diff
@@ -154,130 +150,163 @@ def hashable_deep_deconstruct(cls, obj):
                 ),
             )
             return getattr(obj, "_hashable_deep_deconstructed")
+        elif isinstance(obj, StreamValue):
+            return obj.__class__.__name__
         else:
             return obj
 
+    @staticmethod
+    def get_default_kwargs(block):
+
+        path, args, kwargs = block.deconstruct()
+        signature = inspect.Signature.from_callable(block.__init__)
+        defaults = {
+            k: v.default
+            for k, v in signature.parameters.items()
+            if k not in ("self", "kwargs")
+        }
+        bound = signature.bind(*args, **kwargs).arguments
+        extras = bound.pop("kwargs", {})
+        flat_bound = {**bound, **extras}
+
+        kwargs_dict = {**defaults, **flat_bound}
+
+        for attr in dir(block.meta):
+            if attr.startswith("_"):
+                continue
+            value = getattr(block.meta, attr)
+            kwargs_dict[attr] = value
```

Comment:
Will this overwrite an option that was passed to the block constructor with a value from the Meta class?

Reply:
I think not. https://github.com/wagtail/wagtail/blob/fe3d0c423f87c4892eac62fe8a7f887b00426501/wagtail/blocks/base.py#L77 I believe the options passed to the constructor here overwrite defaults from the Meta class